src/routers/auth.router.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import * as controller from '../controllers/auth.controller';
+
+export const AuthRouter = express.Router();
+
+/**
```

Comment:
we can come back to this later, but swagger has a good amount more we can use to document (error codes etc)